packages/compile-solidity/index.js

```diff
@@ -84,7 +84,7 @@ const Compile = {
       return { compilations: [] };
     }
 
-    Compile.display(compilationTargets, options);
+    displaySources({ paths: compilationTargets, options });
```

Comment:
Incorrect function name here, please fix!

Reply:
Ah missed that one. Good catch!